Policies/github-privacy-statement.md

```diff
@@ -25,119 +45,150 @@ If you're **just browsing the website**, we collect the same basic information t
 
-If you are interested in turning on your browser’s privacy and Do Not Track settings, the [Do Not Track](http://donottrack.us/) website has browser-specific instructions.
+#### Tracking and analytics
 
-Please see our section on [email communication](#how-we-communicate-with-you) to learn about our use of pixel tags in marketing emails.
+We use a number of third party analytics and service providers to help us evaluate our users' use of GitHub; compile statistical reports on activity; and improve our content and website performance. We only use these third party analytics providers on certain areas of our website, and all of them have signed data protection agreements with us that limit the type of personal information they can collect and the purpose for which they can process the information. In addition, we use our own internal analytics software to provide features and improve our content and performance.
+
+We do not currently respond to your browser's Do Not Track signal, and we do not permit third parties other than our analytics and service providers to track GitHub users' activity over time on GitHub. We do not track your online browsing activity on other online services over time. 
 
 ### How GitHub secures your information
 
-GitHub takes all measures reasonably necessary to protect User Personal Information from unauthorized access, alteration, or destruction; maintain data accuracy; and help ensure the appropriate use of User Personal Information. We follow generally accepted industry standards to protect the personal information submitted to us, both during transmission and once we receive it.
+GitHub takes all measures reasonably necessary to protect User Personal Information from unauthorized access, alteration, or destruction; maintain data accuracy; and help ensure the appropriate use of User Personal Information. 
 
-No method of transmission, or method of electronic storage, is 100% secure. Therefore, we cannot guarantee its absolute security. For more information, see our [security disclosures](/articles/github-security/).
+GitHub enforces a written security information program. Our program: 
 
-### GitHub's global privacy practices
-**Information that we collect will be stored and processed in the United States** in accordance with this Privacy Statement. However, we understand that we have users from different countries and regions with different privacy expectations, and we try to meet those needs.
+- aligns with industry recognized frameworks;  
+- includes security safeguards reasonably designed to protect the confidentiality, integrity, availability, and resilience of our users' data; 
+- is appropriate to the nature, size, and complexity of GitHub’s business operations; 
+- includes incident response and data breach notification processes; and 
+- complies with applicable information security related laws and regulations in the geographic regions where GitHub does business. 
```

Comment:
it would be nice to explicitly clarify about security-undermining legislation.

Reply:
I mind mostly not that, but crypto or software backdoors or bans on using secure software.